jormungandr/src/network/mod.rs

```diff
@@ -706,6 +706,11 @@ pub async fn bootstrap(
         }
     }
 
+    blockchain
+        .gc(branch.get_ref().await)
+        .await
```

Comment:
Is this meant to get the storage into steady state, so that periodic GC runs only do a little work with small depth to prune?

Reply:
Exactly. I thought this is better to do before we actually start the node operation.